src/winetricks

```diff
@@ -8670,6 +8670,33 @@ load_flash()
 
 #----------------------------------------------------------------
 
+w_metadata galliumnine02 dlls \
+    title="Gallium Nine Standalone (v0.2)" \
+    publisher="Gallium Nine Team" \
+    year="2019" \
+    media="download" \
+    file1="gallium-nine-standalone-v0.2.tar.gz" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d9-nine.dll" \
+    installed_file2="$W_SYSTEM32_DLLS_WIN/ninewinecfg.exe" \
+    homepage="https://github.com/iXit/wine-nine-standalone"
+
+load_galliumnine02()
+{
+    w_download "https://github.com/iXit/wine-nine-standalone/releases/download/v0.2/$file1"
```

Comment:
$file1 can't be used with w_download (it breaks the tests), please use the explicit filename.

Reply:
Ok, done

Reply:
Thanks